docs/deployment/migration-guide.md

```diff
@@ -28,7 +28,7 @@
 * Since Kyuubi 1.9.0, the support of Spark SQL engine for Spark 3.1 is deprecated, and will be removed in the future.
 * Since Kyuubi 1.9.0, the support of Spark extensions for Spark 3.1 is removed, please use Spark 3.2 or higher versions.
 * Since Kyuubi 1.9.0, `kyuubi.frontend.login.timeout`, `kyuubi.frontend.thrift.login.timeout`, `kyuubi.frontend.backoff.slot.length`, `kyuubi.frontend.thrift.backoff.slot.length` are removed.
-* Since Kyuubi 1.9.0, the support of Flink engine for Flink 1.16 is deprecated, and will be removed in the future.
+* Since Kyuubi 1.9.0, the support of Flink engine for Flink 1.16 is deprecated and removed.
```

Comment:
the removal happens in 1.10, please keep this statement and add a new one in 1.9 to 1.10

Reply:
Thank you for reviewing pr! I will improve it.